### DISCUSSION TITLE
Trumbowyg for HTML Image Content

### DISCUSSION THREAD

**Discussion by serkaya06** [OrchardCMS/OrchardCore#15779] [General]
Hello, 
I am using Orchardcore 1.8.2.
When creating a page I use Trumbowyg for HTML content and "Insert Media with url" is checked
However, when adding an image, it is still "[image]team/2.jpg[/image]"
Is there any way to add <img src="" / >?

Thank you.

**Comment by 13Groszy** [comment]
Edit you trumbowyg code and add button:

btns: [
   ["image"]
]

and add button definition for proper display:

btnsDef: {
   image: {
            dropdown: ["insertImage", "base64", "noembed"],
            ico: "insertImage"
        },
}
